integration-test/test-pages/broker-protection/actions/click-parent.json

```diff
@@ -0,0 +1,42 @@
+{
+  "state": {
+    "action": {
+      "actionType": "click",
+      "elements": [
+        {
+          "type": "button",
+          "selector": ".",
+          "parent": {
+            "profileMatch": {
+              "selector": ".record",
+              "profile": {
+                "name": {
+                  "selector": ".name"
+                },
+                "age": {
+                  "selector": ".age"
+                },
+                "addressCityState": {
+                  "selector": ".address"
+                }
+              }
+            }
+          }
+        }
+      ]
```

Comment:
This is the example where the element matches, and the click should occur on the element - that's why it's `"selector": "."` on line 8